src/utils/json-data.js

```diff
@@ -59,12 +58,15 @@ const buildGlobalDataFromJson = () => {
 
 const convertFileNameToObjectPropertyName = (fileName) => {
   return fileName.split('.')[0]
-    .replace(/\s+/g, '_')     // Replace spaces with underscores
-    .replace(/[^\w-]+/g, '')  // Remove all non-word chars
-    .replace(/-/g, '_')       // Replace all dashes with underscores
-    .replace(/--+/g, '_')     // Replace multiple dashes with single underscore
-    .replace(/^-+/, '')       // Trim - from start of text
-    .replace(/-+$/, '');      // Trim - from end of text;
+    .replace(/^./g, '')        // Remove first character in string
```

Comment:
Why are we removing the first character in the string?  This is causing problems with file names.